init.sql

```diff
@@ -50,7 +50,7 @@ CREATE TABLE IF NOT EXISTS goals (
 
 CREATE TABLE IF NOT EXISTS incomes (
     id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
-    user_id UUID NOT NULL REFERENCES users(id) ON DELETE CASCADE,
+    userid UUID NOT NULL REFERENCES users(id) ON DELETE CASCADE,
```

Comment:
you don't need to change the tables, just map the attributes using SQL queries instead

Reply:
In fact, I changed it not because of the mapper's echo but because the low bar in the middle of the user ID did not seem correct to me.
but you really want to mantain the first ?

Reply:
The truth is that it is a slightly necessary change at this point in the project because it would modify several things that would already be functional, therefore I do not see it as 100% necessary due to a matter of time.